src/utils/ParseRouteUtils.js

```diff
@@ -151,13 +151,8 @@ async function condenseRoute(
     }
 
     // Discard routes where not possible to walk to bus given departure buffer
-    if (departureDelayBuffer) {
-      if (direction.type === 'depart') {
-        const busActualDepartTime = startTime + (direction.delay != null ? direction.delay * 1000 : 0);
-        if (busActualDepartTime < departureTimeNowMs) {
-          return null;
-        }
-      }
+    if (departureDelayBuffer && direction.type === 'depart' && startTime < departureTimeNowMs) {
```

Comment:
I'm not sure why we factor in a `delay` and adjust the departTime for that, and I encountered bugs where it was messing up calculations, so I removed it!